server.js

```diff
@@ -214,7 +214,15 @@ class StoreServer {
       name: key
     }
     try {
-      manifest = await pda.readManifest(archive)
+      
+      // https://javascript.info/promise-api#promise-race
+      // give Paul's Dat API a moment to resolve and win the race
+      // otherwsie, just return the resolved manifest you've already got
+      manifest = Promise.race([
+        await pda.readManifest(archive),
```

Comment:
```suggestion
        pda.readManifest(archive),
```

I think the call to await is blocking execution so the promise is never getting passed to Promise.race :o